README.rst

```diff
@@ -18,6 +18,30 @@ Installation
 
     conda install -c conda-forge nc_time_axis
 
+
+Example Code
+------------
+::
+
+    import nc_time_axis
+	import netcdftime
+	import matplotlib.pyplot as plt
+	import random
```

Comment:
There is a specific order for imports (see the second bullet point in the [pep8 section on imports](https://www.python.org/dev/peps/pep-0008/#imports)), and it helps to alphabetise the imports in each group.

`random` is a standard library and the other imports are "related third part".